privacy_gdpr/tests/test_privacy_gdpr.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import SingleTransactionCase
+from ..wizards.res_partner_gdpr import FIELDS_GDPR
+
+CLEANUP_STRING = '*** GDPR removed ***'
```

Comment:
Now this string is hardcoded English non-translatable text. It might be better to add a simple method to the wizard model like:
```
def get_remove_text(self):
    return _('*** GDPR removed ***')
```
Then use this method both in the wizard itself, and for the test. So you support i18n and at the same time avoid defining the string text in two places.